src/state/themes-mixin.ts

```diff
@@ -88,9 +88,15 @@ export default <T extends Constructor<HassBaseEl>>(superClass: T) =>
       }
 
       const themeMeta = document.querySelector("meta[name=theme-color]");
-      const headerColor = getComputedStyle(
-        document.documentElement
-      ).getPropertyValue("--app-header-background-color");
+      const computedStyles = getComputedStyle(document.documentElement);
+      const headerColor = computedStyles.getPropertyValue(
+        "--app-header-background-color"
+      );
+
+      document.documentElement.style.backgroundColor = computedStyles.getPropertyValue(
+        "--primary-background-color"
+      );
+
```

Comment:
Should we add primary text color here as well?
Should fix #6814

Reply:
Help me, why is that happening now? The theme is loaded in onboarding right?

Reply:
For me, I have seen it when I have set dark for localhost then spin up a fresh dev container.
I tested it, and adding primary-text-color as well, does indeed fix that issue.

Reply:
I would like to know the source, can you reproduce it and check for errors? It is working fine here...

Reply:
![image](https://user-images.githubusercontent.com/5662298/92405630-4ec0af00-f136-11ea-9a8a-d1af41d04d03.png)

Reply:
Let's move it to a new PR